tests/main/views/test_buyers.py

```diff
@@ -566,20 +566,11 @@ def test_expired_framework_post_edit_brief_question(self):
             framework_status='expired'
         )
 
-    def test_expired_framework_post_edit_brief_question(self):
-        data = {"required1": True}
-        self._load_page(
-            "/digital-outcomes/1234/edit/section-1/required1",
-            404,
-            method='post',
-            data=data,
-            framework_status='expired'
-        )
-
-    def test_expired_framework_post_edit_brief_question(self):
+    @pytest.mark.parametrize("framework", ('digital-outcomes', 'digital-specialists'))
```

Comment:
`framework` is probably not the right word to use here; I would recommend using `lot_slug`.